src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const daycost = 40;
+  let total = 0;
+
+  if (days >= 7) {
+    total = days * daycost - 50;
+  } else if (days >= 3) {
+    total = days * daycost - 20;
+  } else {
+    total = +days * daycost;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return days * daycost - 50;
  }
  
  if (days >= 3) {
    return days * daycost - 20;
  } 
  
  return days * costPerDay;
```
використовуй camelCase

Reply:
Якщо ти не проти, я не вводив нову змінну costPerDay, але свою стару змінну dayCost записав в camelCase стилі.